CHANGELOG.md

```diff
@@ -40,6 +40,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [v8.0.0](https://github.com/Stride-Labs/stride/releases/tag/v8.0.0) - 2023-03-23
+### On-Chain changes
+1. v8 Upgrade Handler (incl. adding evmos airdrop and reseting existing airdrops) ([]())
+2. Add Multiple Validators in One Transaction ([]())
+
+### Off-Chain changes
```

Comment:
I think 1 and 4 are on chain changes right?

Or at least the way I've thought about it - On-Chain changes are the changes that change the stride binary (even if there's no stateful changes).